### DISCUSSION TITLE
Reduced diagnostics in PICMI

### DISCUSSION THREAD

**Discussion by pordyna** [BLAST-WarpX/warpx#4685] [Simulation Inputs] [answered]
Does the PICMI interface support reduced diagnostics?

**Comment by ax3l** [comment, marked as answer]
Thanks for the question!

Yes,
https://github.com/ECP-WarpX/WarpX/blob/24.02/Python/pywarpx/picmi.py#L2709

but we forgot to link them in the docs :sweat_smile: 
https://warpx.readthedocs.io/en/latest/usage/python.html#pywarpx.picmi.ParticleDiagnostic

Documentation PR coming via:
https://github.com/ECP-WarpX/WarpX/pull/4693

**Reply by pordyna** [reply]
Oh good to now, thanks!
